10_seawater_intrusion/seawater_intrusion.cfg

```diff
@@ -9,4 +9,8 @@ pressure = 1.e-4 relative
 total_component_concentration = 1.e-4 relative
 timeout = 200
 
-
+[integrated_hydrology_swi_flat]
```

Comment:
If these tests will now pass CI on master, please add them to the "testing" suite.  This adds them to the list of tests that CI runs.  

At the top of the file:
```
[suites]
testing = henry_problem_mfd integrated_hydrology_swi_flat
```